### PR TITLE
Multiple simplifications and optimizations

### PR DESCRIPTION
**Pull request by Prototyx55**

### CODE REVIEW

**Review by memetrollsXD — COMMENTED** [review]
???

**Comment by Young-Alpha** [comment]
no idea why it was "global::System.String" probably your IDE having a fucking stoke.

**Comment by Young-Alpha** [comment]
the new syntax of CurrentOperation switch made me feel a kind of way.

**Comment by nitrog0d** [comment]
:NotLikeThis: change it to "var" or "string" what yall doing bro 😭😭😭

File: OperationHandler.cs [hunk 1]
```diff
@@ -117,8 +104,7 @@ internal static void Automated_Install(string destination, string selected_versi
                         }
                     });
                 }
-                string proxy_path = null;
-                if (GetExistingProxyPath(destination, out proxy_path))
+                if (GetExistingProxyPath(destination, out String proxy_path))
```
Comment (DubyaDude) [inline OperationHandler.cs:107]: Please change to lowercase `string`

File: OperationHandler.cs [hunk 2]
```diff
@@ -244,8 +230,7 @@ internal static void ManualZip_Install(string zip_path, string destination)
...
```
Comment (DubyaDude) [inline OperationHandler.cs:233]: Please change to lowercase `string`

File: OperationHandler.cs [hunk 3]
```diff
@@ -347,8 +332,7 @@ internal static void Uninstall(string destination)
                     });
                 }
 
-                string proxy_path = null;
-                if (GetExistingProxyPath(destination, out proxy_path))
+                if (GetExistingProxyPath(destination, out String proxy_path))
```
Comment (DubyaDude) [inline OperationHandler.cs:335]: Please change to lowercase `string`

File: OperationHandler.cs (hunk 2 above)
Comment (Young-Alpha) [reply OperationHandler.cs:233]: mb didnt notice. 1 second

**Comment by Young-Alpha** [comment]
actually got a skill issue

File: OperationHandler.cs [hunk 4]
```diff
@@ -31,27 +31,14 @@ internal static string CurrentOperationName
             get
             {
                 string opname = null;
-                switch (CurrentOperation)
+                opname = CurrentOperation switch
```
Comment (DubyaDude) [inline OperationHandler.cs:34]: Simplify to
```
string opname = CurrentOperation switch
```

**Comment by DubyaDude** [comment]
LGTM

**Pull request closed without merging**